tests/jballerina-unit-test/src/test/resources/test-src/annotations/annot_attachments_negative.bal

```diff
@@ -975,3 +975,7 @@ type F4 record {|
 type F5 record {|
     int x;
 |};
+
+function testInvalidAnnotationAttachmentOnField() {
+    [@UndefinedAnnotation int, int] [first, second] = [1, 2];
```

Comment:
Don't we need to add a test for a nested tuple also?
`[[@UndefinedAnnotation int, int], int, [int]];`